tests/test_config.py

```diff
@@ -614,6 +630,33 @@ def test_client_options(self):
         conn = connector.Connector.from_config(config_obj)
         self.assertEqual(50, conn.doc_managers[0].mongo.max_pool_size)
 
+    def test_ssl_options(self):
+        config_def = {
+            'mainAddress': 'localhost:27017',
+            'oplogFile': from_here('lib', 'dummy.timestamp'),
+            'ssl': {
+                'sslCertfile': 'certfile.pem',
+                'sslKeyfile': 'certfile.key',
+                'sslCACerts': 'ca.pem'
+            },
+        }
+        for cert_policy, expected_ssl_cert_req in [
+                ('ignored',  ssl.CERT_NONE),
+                ('optional', ssl.CERT_OPTIONAL),
```

Comment:
Though technically that is true, MongoDB doesn't support aNULL ciphers so it's a moot point.

In case you're wondering, CERT_OPTIONAL maps to SSL_VERIFY_PEER, which you can read about here:

https://www.openssl.org/docs/man1.0.1/ssl/SSL_CTX_set_verify.html